mopidy/core/playback.py

```diff
@@ -30,6 +29,9 @@ def __init__(self, audio, backends, core):
         self._last_position = None
         self._previous = False
 
+        self._start_at_pos = None
```

Comment:
s/pos/position/

We generally don't abbreviate variable names, and there's an example three lines up of the same word not being abbreviated.